### DISCUSSION TITLE
Is there minipsf support?

### DISCUSSION THREAD

**Discussion by Miepee** [vgmtrans/vgmtrans#276] [Music formats]
Trying to open a minipsf file from FF7 (got them [here](https://www.zophar.net/music/playstation-psf/final-fantasy-vii)), just results in the following error messages:
```
[RawFile] File C:/Users/me/Desktop/Final Fantasy 7.psflib could not be opened
[PSF1Loader] Unable to open lib file.
```
Not sure if this is just an unsupported file format, or something else.

**Comment by sykhro** [comment]
What build are you using? Please note that you also need the .psflib file to be present in the same folder.

**Comment by Miepee** [comment]
Using the build from commit `9b43626`. And yep, I missed the .psflib file. Thanks!
